### ISSUE TITLE
OSU Navbar: Focus indicators not visible

### ISSUE THREAD

**Issue by McManning** [osuresearch/ui#93]
The focus indicators for the icons in OSU navbar in the header are not visibly at the icons but off to the left side. 
•	[WCAG 2.4.7 Focus Visible](https://www.w3.org/WAI/WCAG21/Understanding/focus-visible.html)
•	Recommendation: Focus indicators should be located at the element.

For context:

![image](https://github.com/osuresearch/ui/assets/1057607/850760c2-1ec8-4d6c-972c-d74296262c66)

**Comment by helloLilyX** [comment]
I think that might be a Chrome issue. I couldn't replicate the error in Safari.

![Screenshot 2023-07-14 at 2 42 20 PM](https://github.com/osuresearch/ui/assets/122384717/f863027e-01d8-43c8-95ce-ccd203abb0ee)

**Comment by McManning** [comment]
I wasn't able to replicate this on Firefox 115.0.2 (64-bit) or Chrome Version 114.0.5735.199. Closing this one
